doc/tutorials/content/sources/passthrough/passthrough.cpp

```diff
@@ -31,7 +31,6 @@ int
   pass.setInputCloud (cloud);
   pass.setFilterFieldName ("z");
   pass.setFilterLimits (0.0, 1.0);
-  //pass.setFilterLimitsNegative (true);
```

Comment:
Please don't simply remove this, instead replace with `setNegative`

Reply:
I thought it was an outdated comment from the context and it did not matter if removed or replaced. Could you explain why it is needed in general because i still cannot make the connection?

Reply:
See [this tutorial](https://pcl.readthedocs.io/projects/tutorials/en/master/passthrough.html) at the very bottom. Also, adding or removing lines in tutorial code is a bit tricky because the tutorials refer to the code by line numbers in the .rst files.
If you want to signal that a PR is not yet ready and we should wait with the reviews, you can mark it as a draft PR.